DAQ_System/Source/Adapter/LinearPotentiometer/linear_potentiometer.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+* Linear Potentiometer
+* Author:        Manuel DJC
+* Modified By:   Cristian Cruz
+*              
+* Email:         Manuel.DeJesusContreras@UTDallas.edu
+*                cris14.alex@gmail.com
+* 
+* (c) 2022 Dallas Formula Racing - Embedded Firmware Team
+* Formula SAE International Collegiate Chapter
+* GPL-3.0 License
+*/
+
+#include "AnalogIn.h"
+#include "mbed.h"
```

Comment:
These includes may be unnecessary here, as they are included in the corresponding header file